### PR TITLE
multibuildpack support

### PR DESCRIPTION
**Pull request by hone**
Parse the `exports` file produced by multibuildpack to make them available to subsequent builds.

### CODE REVIEW

File: bin/build [hunk 1]
```diff
@@ -23,5 +23,11 @@ mkdir -p "${profile_dir}/profile.d"
 cp .profile.d/* "${profile_dir}/profile.d/"
 echo "launch = true" > ${layers_dir}/profile.toml
 
+if [ -f "{$target_dir}/export" ]; then
```
Comment (jkutner) [inline bin/build]: should this be `"{$target_dir}/exports"`

**Comment by jkutner** [comment]
The CI failure is because the tests are expecting the process types to come back in a specific order in the array, but apparently the order is non-deterministic. So you can ignore that

File: bin/build (hunk 1 above)
Comment (hone) [reply bin/build]: thanks!

**Comment by jkutner** [comment]
@hone @elbandito need a review on 5ab805c

File: bin/build [hunk 2]
```diff
@@ -21,7 +21,13 @@ echo "cache = true" > ${layers_dir}/shim.toml
 profile_dir="${layers_dir}/profile"
 mkdir -p "${profile_dir}/profile.d"
 cp .profile.d/* "${profile_dir}/profile.d/"
-echo "launch = true" > ${layers_dir}/profile.toml
+echo "launch = true" > "${profile_dir}.toml"
```
Comment (hone) [inline bin/build]: How was this every working before?

**Comment by hone** [comment]
apparently, I can't approve my own PR. This looks good to me.

**Pull request merged**